lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -493,6 +493,40 @@ module.exports = class MethodMappings {
           };
         },
 
+
+        getElementComputedRole(webElementOrId) {
+          const promise = (async () => {
+            const id = await webElementOrId.getId();
+
+            return `/element/${id}/computedrole`;
+          })();
+          promise.resolveEarly = true;
+
+          return promise;
+        },
+
+        getElementComputedLabel(webElementOrId) {
+          const promise = (async () => {
+            const id = await webElementOrId.getId();
+
+            return `/element/${id}/computedlabel`;
+          })();
+          promise.resolveEarly = true;
+
+          return promise;
+        },
+
+        async isElementActive(webElementOrId) {
+          const element = this.getWebElement(webElementOrId);
+
+          const isActive = await this.driver.executeScript(`
+            const el = arguments[0];
+            return el === document.activeElement;
+          `, element);
```

Comment:
Shouldn't we use [this endpoint](https://w3c.github.io/webdriver/#get-active-element) to get the active element?

Reply:
this actually returns the active element, so we won't be able to get the element here to check if the element is that element only.